lib/internal/modules/esm/loader.js

```diff
@@ -565,29 +611,20 @@ class ESMLoader {
           new URL(nextUrl);
         } catch {
           throw new ERR_INVALID_ARG_VALUE(
-            `${hookErrIdentifier} nextLoad(url)`,
+            `${hookErrIdentifier}s nextLoad(url)`,
```

Comment:
Do you mean apostrophe s? Or maybe it’s better without the possessive?

Reply:
`hookErrIdentifier` already ends with `'`. I didn't add an apostrophe because a somewhat recent PR converted proper apostrophes into single-quote marks. The output here looks something like:

```
The ESM custom loader "file//foo.mjs 'load's  nextLoad(url)" must be a url string but got an instance of Array.
```

Reply:
Maybe just add the word `hook`?
```suggestion
            `${hookErrIdentifier} hook's nextLoad(url)`,
```

`TypeError [ERR_INVALID_ARG_TYPE]: The "file:///foo.mjs 'resolve' hook's nextResolve(specifier)" property`

Reply:
Good idea! I tweaked it a little more:

`TypeError [ERR_INVALID_ARG_TYPE]: The "file:///foo.mjs 'resolve' hook's nextResolve() specifier" property must be of type string. Received an instance of Array`